pkg/cmd/tests/tests.go

```diff
@@ -77,6 +78,7 @@ func NewTestsCommand(streams genericclioptions.IOStreams) *cobra.Command {
 	}
 
 	userAgent := "scylla-operator-e2e"
+	cmd.AddCommand(versioncmd.NewVersionCmd(streams))
```

Comment:
how about we set cmd.Version to have `--version` argument on root command? This PR would be `+4/-0`.

Reply:
That would mean the command without any args would have to tell you that you need to specify either `--version` flag or a sub-command which is inconsistent, so I very much prefer the separate sub-command. This is also an analogy to `kubectl version`.

Reply:
also the sub-command is more fitting to doing additional flags like `--json`

Reply:
I agree with Tomas, `$command version` is more idiomatic in the entire go-cli landscape and I'd expect our executables to do that rather than the flag